<launch>
    <!-- Lane Keeping System -->
    <node name="Traffic_Sign_Depth_Estimation" pkg="traffic_sign_depth_estimation" type="traffic_sign_depth_estimation_node" output="screen"/>
    <param name="config_path" type="str" value="$(find lane_keeping_system)/config/config.yaml"/>
    
    <!-- USB Cam -->
    <include file="$(find usb_cam)/launch/usb_cam-noviewer.launch"/>

     <!-- Xycar Lidar -->
    <include file="$(find xycar_lidar)/launch/lidar_noviewer.launch"/>

    <!-- Yolo Module -->
    <include file="$(find yolov3_trt_ros)/launch/yolov3_trt_ros_detection.launch"/>
</launch>
